genie_registry/clinical.py

```diff
@@ -174,7 +174,27 @@ def _check_int_year_consistency(
 
     return ""
 
+def _check_year_death_validity(clinicaldf: pd.DataFrame) -> str:
+    """
+    Check if YEAR_DEATH >= YEAR_CONTACT.
+    When YEAR_DEATH is known, YEAR_CONTACT should = YEAR_DEATH.
 
+    Args:
+        clinicaldf: Clinical Data Frame
+        
+    Returns:
+        Error message if YEAR_DEATH if invalid
+    """    
+    error = ""
+    try:
+        check_result = np.where((pd.isna(clinicaldf['YEAR_DEATH']) | pd.isna(clinicaldf['YEAR_CONTACT'])), 'N/A', clinicaldf['YEAR_DEATH'] >= clinicaldf['YEAR_CONTACT'])
```

Comment:
Sorry, this is not explicitly mentioned in the ticket but for all validation tickets, let's try to set up the validation for future row-based validation work. So having this give you the row indices (i've been returning pandas Index) that `YEAR_DEATH` < `YEAR_CONTACT`. And then your check will be on whether it's empty row indices or not

Hope that makes sense.

Reply:
Just added the line of code to show the index of the rows of YEAR_DEATH < YEAR_CONTACT for future use.

Reply:
Thanks! I think you can still keep your check or check based on the row indices. It's enough to have the index of rows saved for later so thanks for adding that!